qa/automation/CareerPortal.Tests.Scenarios/Savostin/buttons.feature

```diff
@@ -0,0 +1,52 @@
+Feature: 1.0.0 buttons lead to correct pages
+Background: 
+	Given "Career Quantori" page is opened
+
+Scenario Outline: 1.0.0_01 All buttons of header lead to correct pages
+	When user clicks on <button>
+	Then <page> is opened
+
+	Examples:
+	|button            |page                                 |
```

Comment:
From my point of view, one more button should be added here: Quantori logo in the left upper corner

Reply:
Done) You can find it in buttons_of_Career_Quantori_page.feature 1.0.0_01